### PR TITLE
Process opening refactor

### PR DESCRIPTION
**Pull request by chicoxyzzy**

### CODE REVIEW

File: src/desktop/main.cc [hunk 1]
```diff
@@ -500,61 +535,61 @@ MAIN {
...
     }
 
-    if (cmd.name == "inspect") {
+    if (message.name == "inspect") {
       window->showInspector();
       return;
     }
 
-    if (cmd.name == "background") {
+    if (message.name == "background") {
```
Comment (heapwolf) [inline src/desktop/main.cc:548]: we could make this message name less ambiguous while we're here

**Review by heapwolf — APPROVED** [review]

File: src/desktop/main.cc (hunk 1 above)
Comment (chicoxyzzy) [reply src/desktop/main.cc:548]: There was two different `cmd` variables:
- `<os>_cmd` string
- command from webvew process or backend process
I think that message is a good name for this IPC thing, but I'm open to other alternatives

**Comment by chicoxyzzy** [comment]
CI is complaining on nodejs desktop example from `socket-sdk-examples` this is intended as bootstrapping has changed order of loading. I can update examples repo and run CI agains new branch. We will need to merge examples update and then change branch back to master and merge this PR

File: src/desktop/main.cc (hunk 1 above)
Comment (chicoxyzzy) [reply src/desktop/main.cc:548]: I like `ipcMessage` as well

**Review by jwerle — APPROVED** [review]

**Comment by chicoxyzzy** [comment]
the blocker is https://github.com/socketsupply/socket-sdk-examples/pull/36 and changing examples repo back to master

**Pull request merged**